src/lib/timerStateMachine.ts

```
@@ -0,0 +1,288 @@
+
+    switch (event.type) {
+      case 'START':
+        this.updateContext({
+          state: 'running',
+          startTime: now,
+          pausedTime: null
+        });
+        break;
+
+      case 'PAUSE':
+        const elapsed = this.getElapsedTime();
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)